WeakAuras/WeakAuras.lua

```diff
@@ -3143,6 +3142,74 @@ function WeakAuras.CancelAnimation(region, resetPos, resetAlpha, resetScale, res
   end
 end
 
+function WeakAuras.GenerateDataStub(regionType, id)
```

Comment:
Given that two of the usages are read-only, wouldn't it better to have that table constant somewhere? The one place that needs a copy can make a copy.